core/class/htmldisplay.class.php

```diff
@@ -22,7 +22,7 @@
 class htmldisplay extends eqLogic {
   /*     * *************************Attributs****************************** */
 
-
+ protected static $_widgetPossibility = array('custom' => true);
```

Comment:
we can argue on the fact that property should be private or protected but today $_widgetPossibility is accessed directly by the core, so outside of the class; I would suggest to let it public as it is in plugin-template